t/ui/18-tests-details.t

```diff
@@ -151,7 +151,8 @@ subtest 'show job modules execution time' => sub {
 subtest 'displaying image result with candidates' => sub {
     $driver->find_element('[href="#step/bootloader/1"]')->click();
     wait_for_ajax;
-    is_deeply(find_candidate_needles, {'inst-bootmenu' => []}, 'correct tags displayed');
+    my $needles = find_candidate_needles;
+    is_deeply(keys %$needles, @{['inst-bootmenu']}, 'correct tags displayed');
```

Comment:
```suggestion
   is_deeply(find_candidate_needles, {'inst-bootmenu' => []}, 'correct tags displayed') or diag explain $needles;
```

Reply:
Also not sure what `@{['inst-bootmenu']}` is supposed to achieve. It's creating an array ref and dereferences it immediately.

Reply:
Well, you can see in the description what the result is when it fails. Without the `@{}` all you get is `ARRAY(0xaaab25f52718`

Reply:
`@{['inst-bootmenu']}` is just the same as `'inst-bootmenu'`.

Reply:
What happens if `keys %$needles` returns more than one key? This usage of `is_deeply` wrong.
Say if it has three keys, then your code would be equivalent to:
```
    is_deeply('key1', 'key2', 'key3', 'inst-bootmenu', 'correct tags displayed');
```
What should that do?